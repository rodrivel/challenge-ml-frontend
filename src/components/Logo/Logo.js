import React from 'react';
import PropTypes from 'prop-types';
import logo1x from '../../assets/Logo_ML.png';
import logo2x from '../../assets/Logo_ML@2x.png.png';
import { Link } from 'react-router-dom';
import cx from './Logo.module.scss';

const logo = (props) => {
  return (
    <Link to="/" className={cx.Logo__Link} id='logo-link'>
      <img width={props.width || 106} src={logo1x} srcSet={`${logo1x} 53w, ${logo2x} 106w`} alt="Logo Mercadolibre"/>
    </Link>
  );
};

logo.propTypes = {
  width: PropTypes.number
};

export default logo;