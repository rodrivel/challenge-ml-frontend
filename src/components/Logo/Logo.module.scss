.Logo__Link {
  display: flex;
  align-items: center;
}