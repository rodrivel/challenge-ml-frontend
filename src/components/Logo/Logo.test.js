import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

it('expect to render Logo component', () => {
  expect(shallow(<Logo />)).toMatchSnapshot();
});
