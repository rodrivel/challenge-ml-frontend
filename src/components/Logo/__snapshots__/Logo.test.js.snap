// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Logo component 1`] = `ShallowWrapper {}`;
